### ISSUE TITLE
TVirtualTreeAccessibility.accSelect() should handle the flag SELFLAG_EXTENDSELECTION

### ISSUE THREAD

**Issue by joachimmarder** [JAM-Software/Virtual-TreeView#693]
`TVirtualTreeAccessibility.accSelect()` should handle the flag `SELFLAG_EXTENDSELECTION`. This is useful for screenreaders and automatic testing tools.

To achieve this, `TVirtualTreeAccessibility` should have access to `HandleClickSelection()`.
